.packit.yaml

```diff
@@ -15,34 +15,41 @@ copy_upstream_release_description: true
 
 srpm_build_deps:
     - cargo
+    - openssl-devel
 
 actions:
     create-archive:
-        - bash -c "sed -i -r  \"s/^Version:.*/Version:\        ${PACKIT_PROJECT_VERSION}/\" fido-device-onboard.spec"
-        - bash -c "sed -i '/Source1/d ; /^# See make-vendored-tarfile.sh in upstream repo/d ;' fido-device-onboard.spec"
         - bash -c "cp ./patches/0001-Revert-chore-use-git-fork-for-aws-nitro-enclaves-cos.patch ."
         - bash -c "git archive --prefix=fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}/ --format=tar HEAD > fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar"
+        - bash -c "./make-vendored-tarfile.sh ${PACKIT_PROJECT_VERSION}"
         - bash -c "tar -xvf fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar"
         - bash -c "ls -1 ./fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar"
-    fix-spec-file:
-        - "cat fido-device-onboard.spec"
 
 jobs:
-    - job: copr_build
+    - &fdo_copr_build
+      job: copr_build
       trigger: pull_request
       targets:
-          - fedora-development-aarch64
-          - fedora-development
-          - fedora-latest
-          - fedora-latest-aarch64
-    - job: copr_build
+          epel-9: {}
+          fedora-latest-stable: {}
+          fedora-developement: {}
+          fedora-eln:
+              additional_repos:
+                  - https://kojipkgs.fedoraproject.org/repos/eln-build/latest/$basearch/
+
+    - job: tests
+      trigger: pull_request
+      identifier: onboarding
+      fmf_path: test/fmf
+      tmt_plan: plans/onboarding
```

Comment:
The tmt plan defined here, so the rpmlint will not be run in this case.

Reply:
I removed it from the source code, we will probably add more tests in the future.